src/app/clusters/ota-provider/ota-provider.cpp

```diff
@@ -168,10 +168,12 @@ bool emberAfOtaSoftwareUpdateProviderClusterQueryImageCallback(EndpointId endpoi
     ChipLogDetail(Zcl, "OTA Provider received QueryImage");
 
     // TODO: (#7112) change location size checking once CHAR_STRING is supported
-    const uint8_t locationLen = emberAfStringLength(location);
+    // The following commented code does not work because location is being treated as uint8_t* and this leads to always sending a
```

Comment:
The code is not commented, that I can see....  Unless you mean that this is the comment or something like that?  In which case probably clearer to say "The code using strlen"... or something?

Reply:
this was a mistake, I meant to remove this. Thanks for catching it.